addons/hr_expense/static/src/views/expense_list_view.js

```diff
@@ -0,0 +1,49 @@
+/** @odoo-module */
+
+import { registry } from "@web/core/registry";
+
+import { ListRenderer } from "@web/views/list/list_renderer";
+import { X2ManyField, x2ManyField } from "@web/views/fields/x2many/x2many_field";
+
+import { useService } from "@web/core/utils/hooks";
+
+export class ExpenseListRenderer extends ListRenderer {
```

Comment:
Are you aware there is already an `ExpenseListRenderer` class in list.js?
Maybe we should rename this one to avoid confusion?